content/en/docs/instrumentation/java/libraries.md

```diff
@@ -4,4 +4,22 @@ linkTitle: Libraries
 weight: 40
 ---
 
-_to be done_
+When you develop an app, you use third-party libraries and frameworks to
+accelerate your work and avoid duplicated efforts. If you instrument your app
+with OpenTelemetry, you don't want to spend additional time on manually adding
+traces, logs, and metrics to those libraries and frameworks.
+
+Use libraries that come with OpenTelemetry support natively or an
+[Instrumentation Library](/docs/concepts/instrumentation/libraries/) to generate
+telemetry data for a library or framework.
+
+The Java agent for automatic instrumentation includes instrumentation libraries
+for many common Java frameworks. All instrumentations are turned on by default.
+If you need to turn off certain instrumentation libraries, you can
+[suppress them](../automatic/agent-config/#suppressing-specific-auto-instrumentation).
+
+If you use [manual instrumentation](../manual) for your code, you can leverage
+some instrumentation libraries for your dependencies standalone. To find out,
+which standalone instrumentation libraries are available, take a look at
+[this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
```

Comment:
I wish we migrated that list over to this repo at some point...

Reply:
+1, yes this page contains a lot of value that could/should live on this particular page. But we can migrate gradually, let's have a basic version of this page first, such that people find the right info, next we can consider moving material.

Reply:
Yeah, definitely. I think this PR is a good first step towards that goal.